### ISSUE TITLE
No information displayed on terminal

### ISSUE THREAD

**Issue by ashaykolhe** [FormidableLabs/webpack-dashboard#98]
> Please provide a description of the bug / issue, and provide the details below:

a. There is no information displayed on the terminal. Please check the attached screenshot.
b. The dev-server server isn't starting either. I guess probably due to some error while dashboard startup.

====================================================================

###### If the issue is visual, please provide screenshots here

====================================================================

###### Steps to reproduce the problem

====================================================================

###### Please provide a gist of relevant files

1. package.json (specifically the script you are using to start the dashboard)
`{
  "name": "sc-react-redux",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "start": "webpack-dashboard -- webpack-dev-server --progress --colors --hot --config ./webpack.config.js",
    "test": "mocha --compilers js:babel-core/register --require ./test/setup.js src/**/*spec.js",
    "test:watch": "npm run test -- --watch"
  },
  "keywords": [],
  "author": "",
  "license": "ISC",
  "babel": {
    "presets": [
      "es2015",
      "react",
      "stage-2"
    ]
  },
  "devDependencies": {
    "babel-core": "^6.9.1",
    "babel-loader": "^6.2.4",
    "babel-preset-es2015": "^6.9.0",
    "babel-preset-react": "^6.5.0",
    "babel-preset-stage-2": "^6.5.0",
    "chai": "^3.5.0",
    "enzyme": "^2.3.0",
    "exports-loader": "^0.6.3",
    "imports-loader": "^0.6.5",
    "jsdom": "^9.2.1",
    "mocha": "^2.5.3",
    "react-addons-test-utils": "^15.1.0",
    "react-hot-loader": "^1.3.0",
    "webpack": "^1.13.1",
    "webpack-dashboard": "^0.1.7",
    "webpack-dev-server": "^1.14.1"
  },
  "dependencies": {
    "react": "^15.1.0",
    "react-dom": "^15.1.0",
    "react-redux": "^4.4.5",
    "react-router": "^2.4.1",
    "react-router-redux": "^4.0.5",
    "redux": "^3.5.2",
    "redux-logger": "^2.6.1",
    "redux-thunk": "^2.1.0",
    "soundcloud": "^3.1.2",
    "whatwg-fetch": "^1.0.0"
  }
}
`

2. webpack.config.js

`var webpack = require('webpack');
var DashboardPlugin = require('webpack-dashboard/plugin');

module.exports = {
  entry: [
    'webpack-dev-server/client?http://localhost:8080',
    'webpack/hot/only-dev-server',
    './src/index.js'
  ],
  output: {
    path: __dirname + '/dist',
    publicPath: '/',
    filename: 'bundle.js'
  },
  module: {
    loaders: [{
      test: /\.js?$/,
      exclude: /node_modules/,
      loader: 'react-hot!babel'
    }]
  },
  resolve: {
    extensions: ['', '.js', '.jsx']
  },
  devServer: {
    contentBase: './dist',
    hot : true,
    quiet : true,
    historyApiFallback: true
  },
  plugins: [
    new webpack.ProvidePlugin({
      'fetch': 'imports?this=>global!exports?global.fetch!whatwg-fetch'
    }),
    new DashboardPlugin()
  ]
};`
3. index.js (Your express based dev server, if applicable) -- not using express
![dashboard](https://cloud.githubusercontent.com/assets/1070123/17904153/d187f916-698b-11e6-875d-d6d060e58798.jpg)

**Comment by luisrudge** [comment]
I have the same problem

**Comment by mcMickJuice** [comment]
In webpack-dashboard/bin/webpack-dashboard.js, in the stdio array passed to child process, change "ipc" to null (or comment out stdout altogether). Does it work?

```
var child = spawn(command, args, {
  env: env,
  stdio: [null, null, null, null],
  detached: false
});
```

**Comment by kenwheeler** [comment]
Is the default port open?

**Comment by ashaykolhe** [comment]
@kenwheeler I am using 8080 port for dev server which is open.

@mcMickJuice changing "ipc" to null worked. Thanks.

**Comment by kenwheeler** [comment]
Fixed this on master, released 0.1.8
